src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -139,6 +139,16 @@
     short-summary: Manage file storage directory metadata.
 """
 
+helps['storage directory list'] = """
+    type: command
+    short-summary: List directories in the specified share.
+    parameters:
+        - name: --num-results
+          type: integer
+          short-summary: Specifies the maximum number of directories to return. The default value is
+                         5000. The value cannot be less or equal to zero.
```

Comment:
Do you like the idea to hide the option on both `az storage file list` and `az storage directory list` until we have a good paging story?